tree/dataframe/test/dataframe_regression_tests.hxx

```diff
@@ -106,3 +109,40 @@ TEST(TEST_CATEGORY, UniqueEntryNumbers)
 
    gSystem->Unlink(fname);
 }
+
+// ROOT-9731
+TEST(TEST_CATEGORY, ReadWriteTVector3)
+{
+   const std::string filename = "readwritetvector3.root";
+   {
+      TFile tfile(filename.c_str(), "recreate");
+      TTree tree("t", "t");
+      auto a = TVector3();
+      tree.Branch("a", &a); // TVector3 as TBranchElement
+      auto c = new TVector3();
```

Comment:
Leaks. Is only relevant if we use the test to track memory leaks, though.

Reply:
thanks! will fix